audformat/core/index.py

```diff
@@ -103,6 +113,23 @@ def segmented_index(
     Raises:
         ValueError: if ``files``, ``start`` and ``ends`` differ in size
 
+    Example:
+        >>> segmented_index(['a.wav', 'b.wav'])
+        MultiIndex([('a.wav', '0 days', NaT),
+                    ('b.wav', '0 days', NaT)],
+                   names=['file', 'start', 'end'])
+        >>> segmented_index(['a.wav'], [0], [1])
+        MultiIndex([('a.wav', '0 days', '0 days 00:00:00.000000001')],
+                   names=['file', 'start', 'end'])
+        >>> segmented_index(
+        ...     files=['a.wav', 'a.wav'],
+        ...     starts=[t * 10 ** 9 for t in (0, 1)],
```

Comment:
I find this syntax a little bit too complicated for a usage example.

Reply:
Would the following be better:
```python
starts=[f'{t}s' for t in (0, 1)],
```
For me the above line is still easier then using `pd.Timedelta` as I can never remember how to use that one.
The idea behind this line is also to make it easier when https://github.com/audeering/audformat/issues/12 is addressed.

Reply:
What about just `['0s', '1s']`?

Reply:
or `['0s', '1000ms']` to show that other units are supported, too?

Reply:
Yes `['0s', '1s']` it's fine for an example, but in reality you most probably have already a long list with integers or floats and not strings.

Reply:
Ok, let's use `[f'{t}s' for t in (0, 1)]` here and for `ends` use `pd.to_timedelta([1, 2], unit='s')`. Then we show two scalable options.

Reply:
Changed it accordingly.